feat: allow passing down "null" to disable server watcher (#14208)
vitejs/vite: docs/guide/api-javascript.md

   */
  httpServer: http.Server | null
  /**

>    * Chokidar watcher instance. If watcher was disabled in the config,
>    * returns noop event emitter.

Reviewer: ```suggestion
   * Chokidar watcher instance. If `config.server.watch` is set to `null`,
   * returns a noop event emitter.
```

tiny nit
